providers/msGraph/README.md

```diff
@@ -0,0 +1,19 @@
+# Novu MSGraph API Provider
+
+A MSGraph API email provider library for [@novu/stateless](https://github.com/novuhq/novu)
```

Comment:
I think I favour the naming of MSGraph rather than MSGraph API due that we use the client package provided by them to connect to their API.